GothicVRProject/GothicVR
# Bugfix/armor misalignment

# To test
1. Load OC NPCs
2. Check if the head is at least close to where it should be. ;-)
3. Especially checking for: Babe, Xardas (Because of the high robe), some other NPCs (where you saw weird head and arms before)
